### PR TITLE
validator table wasn't loading

### PR DESCRIPTION
**Pull request by jbibla**

### CODE REVIEW

File: pages/validators/index.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
       :delegations="delegations"
       :rewards="rewards"
       :search-term="searchTerm ? true : false"
-      :loaded="delegationsLoaded"
+      :loaded="validatorsLoaded"
```
Comment (faboweb) [inline pages/validators/index.vue:29]:
```suggestion
      :loaded="validatorsLoaded && delegationsLoaded"
```

Comment (faboweb) [reply pages/validators/index.vue:29]: is delegationsLoaded not set?

**Comment by faboweb** [comment]
Validator list is actually loading on master https://lunie3.netlify.app/validators

**Review by faboweb — CHANGES_REQUESTED** [review]
Is there even a problem? Live this works.

File: pages/validators/index.vue (hunk 1 above)
Comment (jbibla) [reply pages/validators/index.vue:29]: why should validator table block on loading delegations?

**Comment by jbibla** [comment]
for me it does not load - and on e-money wallet /shrug

**Comment by faboweb** [comment]
Ok I see the issue.

**Pull request merged**